pkg/kubernetes/scripts/projects.js

```diff
@@ -103,6 +112,53 @@
         }
     ])
 
+
+    .controller('UserCtrl', [
+        '$scope',
+        '$routeParams',
+        '$location',
+        'kubeSelect',
+        'kubeLoader',
+        'projectData',
+        'projectActions',
+        'roleActions',
+        'ListingState',
+        function($scope, $routeParams, $location, select, loader, projectData, projectAction, roleActions, ListingState) {
+            loader.watch("users");
+            loader.watch("groups");
+            var user = $routeParams["user"] || "";
+            if (user) {
+
+                var userObj = select().kind("User").name(user);
+                if(!userObj || userObj.length < 1) {
```

Comment:
@petervo Did you build a "not found" pattern that you used in the volume page? It would be good to use that here.